runtime/runtime.go

```diff
@@ -848,6 +865,26 @@ func (rt *Runtime) getWatcher(rootPaths []string) (*fsnotify.Watcher, error) {
 	return watcher, nil
 }
 
+func urlPathToConfigOverride(pathCount int, path string) ([]string, error) {
+	uri, err := url.Parse(path)
+	if err != nil {
+		return nil, err
+	}
+	baseUrl := uri.Scheme + "://" + uri.Host
+	urlPath := uri.Path
+	if uri.RawQuery != "" {
+		path += "?" + uri.RawQuery
+	}
+
+	var override []string
+	override = append(override, fmt.Sprintf("services.cli%d.url=%s", pathCount, baseUrl))
+	override = append(override, fmt.Sprintf("bundles.cli%d.service=cli%d", pathCount, pathCount))
+	override = append(override, fmt.Sprintf("bundles.cli%d.resource=%s", pathCount, urlPath))
+	override = append(override, fmt.Sprintf("bundles.cli%d.persist=true", pathCount))
+
+	return override, nil
```

Comment:
[nit] We could just make it explicit and one-off:

```suggestion
	return []string{
		fmt.Sprintf("services.cli%d.url=%s", pathCount, baseUrl),
		fmt.Sprintf("bundles.cli%d.service=cli%d", pathCount, pathCount),
		fmt.Sprintf("bundles.cli%d.resource=%s", pathCount, urlPath),
		fmt.Sprintf("bundles.cli%d.persist=true", pathCount),
	}, nil
```

or even like this:
```go
return strings.Split(fmt.Sprintf(`services.cli%[1]d.url=%[2]s bundles.cli%[1]d.service=cli%[1]d bundles.cli%[1]d.resource=%[3]s bundles.cli%d.persist=true`, pathCount, baseUrl, urlPath), " ")
```

Reply:
I went with the former.